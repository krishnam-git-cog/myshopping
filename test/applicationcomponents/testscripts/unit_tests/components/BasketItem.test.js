import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import BasketItem from '../../../../../src/components/basketitem/BasketItem';

describe('BasketItem test', () => {
  it("Component - BasketItem should match snapshot",  () => {
    let shallow = createShallow();
    const data = {
      "productImage": "https://place-hold.it/300x500",
      "productCode": "ABC123",
      "productName": "Fitbit",
      "productPrice": "189.00"
    }
    const component = shallow(<BasketItem data={data} />);
    expect(component).toMatchSnapshot();
  });
});