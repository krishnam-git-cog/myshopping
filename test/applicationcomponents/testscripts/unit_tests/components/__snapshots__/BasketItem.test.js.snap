// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasketItem test Component - BasketItem should match snapshot 1`] = `ShallowWrapper {}`;
